lib/stringlib/lua/arith.lua

```diff
@@ -0,0 +1,87 @@
+local n = {}
+local meta = {}
+setmetatable(n, meta)
+meta.__tostring = function() return "<n>" end
+meta.__add = function(x, y)
+    if x == "BOOM" then
+        error(x)
+    end
+    return x
+end
+meta.__idiv = function(x, y)
+    if x == "BOOM" then
+        error(x)
+    end
+    return x
```

Comment:
and here did you mean `x // y`? Guess it doesn't matter too much since I saw we're not actually using them for arithmetic in the tests below.

Reply:
Same as above